### PR TITLE
wording improvements and some ruby fixes

### PR DESCRIPTION
**Pull request by rilian**
1) readme wording improvements
2) rspec imports simplified by using `.rspec`. bonus: now it has colors
3) remove unnecessary `return` operators
4) fix attr_accessor assignment in `Transaction` class
5) few spec example description improvements according to http://betterspecs.org/ suggestions

### CODE REVIEW

**Comment by darthrevan** [comment]
:+1:

**Comment by michaelckelly** [comment]
@rilian - thank you for taking the time to put this together, nice improvements all around!

File: lib/plaid/models/transaction.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def initialize(fields = {})
       @name = fields['name']
       @location = fields['meta'].nil? ? {} : fields['meta']['location']
       @pending = fields['pending']
-      @pendingTransaction = fields['_pendingTransaction']
+      @pending_transaction = fields['_pendingTransaction']
```
Comment (michaelckelly) [inline lib/plaid/models/transaction.rb]: :+1:

**Comment by michaelckelly** [comment]
@dmerrick - any thoughts on this? If not, I say let's get this merged and published as version `1.6.0`!

**Comment by dmerrick** [comment]
Seems reasonable to me!

**Comment by michaelckelly** [comment]
Cool - I'll merge away then. Thanks again for the contribution, @rilian!

**Comment by rilian** [comment]
thanks :)

**Pull request merged**